services/database-analyzer-mongo.js

```diff
@@ -0,0 +1,104 @@
+const _ = require('lodash');
+const logger = require('./logger');
+const P = require('bluebird');
+const { DatabaseAnalyzerError } = require('../utils/errors');
+
+function isUnderscored(fields) {
+  return fields.every(field => field.nameColumn === _.snakeCase(field.nameColumn))
+    && fields.some(field => field.nameColumn.includes('_'));
+}
+
+function analyzeMongoCollection(databaseConnection, collectionName) {
+  return new Promise((resolve, reject) => {
+    databaseConnection.collection(collectionName)
+    /* eslint-disable */
```

Comment:
[Code Style] Do you still need this comment? If so, you should reindent it to make the code easy to read (and like it was before)